packages/@o3r/extractors/src/utils/config-doc.ts

```diff
@@ -138,6 +141,17 @@
   return /@o3rRequired/.test(docText);
 }
 
+/**
+ * Get restriction keys from a given DocComment.
+ *
+ * The restriction keys extracted from @o3rRestrictionKeys tag.
+ * @param docComment The DocComment to get category from
+ */
+export function getRestrictionKeysFromDocText(docComment: string): string[] {
+  return Array.from(docComment.matchAll(/@o3rRestrictionKeys\s+(\w+|(["']).*?\2)$/gm))
```

Comment:
I can do `/@o3rRestrictionKeys\s+["'](\w+)["']$/gm`
but with this one it will validate `@o3rRestrictionKeys "invalid quotes'`
Does anyone has an idea, how to fix this regexp without invalidating the above use case ?